segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -93,14 +95,15 @@ namespace pcl
         * \param depth_dependent
         */
       inline void
-      setDistanceThreshold (float distance_threshold, bool depth_dependent)
+      setDistanceThreshold (const float &distance_threshold,
+                            const bool &depth_dependent)
       {
         distance_threshold_ = distance_threshold;
         depth_dependent_ = depth_dependent;
       }
 
       /** \brief Get the distance threshold in meters (d component of plane equation) between neighboring points, to be considered part of the same plane. */
-      inline float
+      inline const float&
```

Comment:
Again: preventing a copy if you don't desire one. But in the end I'm not sure what exactly happens under the hood, when you create a reference.